.packit.yaml

```diff
@@ -37,15 +37,9 @@ jobs:
   - <<: *copr
     project: qm
     targets:
-      fedora-40-aarch64: {}
-      fedora-40-ppc64le: {}
-      fedora-40-x86_64: {}
-      fedora-39-aarch64: {}
-      fedora-39-ppc64le: {}
-      fedora-39-x86_64: {}
-      fedora-38-aarch64: {}
-      fedora-38-ppc64le: {}
-      fedora-38-x86_64: {}
+      fedora-stable-aarch64: {}
+      fedora-stable-ppc64le: {}
+      fedora-stable-x86_64: {}
       centos-stream-9-aarch64:
         additional_repos:
           - https://dl.fedoraproject.org/pub/epel/9/Everything/aarch64/
```

Comment:
Are you doing this to build explicitly on CentOS Stream 9 and not RHEL 9? Otherwise you could simply use `epel9-aarch64` with no `additional_repos`.

Reply:
@nforro perfect. Would mind to send us another contribution? :)

Reply:
Will do.